test/evergreen.yml

```diff
@@ -1131,18 +1175,6 @@ tasks:
           test_config_filename: configs/hs_cleanup_default.txt
           test_name: hs_cleanup
 
-  - name: cppsuite-burst-inserts-default
-    tags: ["pull_request"]
-    depends_on:
-      - name: compile
-    commands:
-      - func: "fetch artifacts"
-      - func: "cppsuite test"
-        vars:
-          test_config: debug_mode=(cursor_copy=true)
-          test_config_filename: configs/burst_inserts_default.txt
-          test_name: burst_inserts
```

Comment:
I can see that a couple of cppsuite tasks got removed, is that a temporary thing? I can't see a replacement anywhere but I might be missing it in the monster diff for this file.

Reply:
It was a mistake. Great catch.

Reply:
Re-reviewing, I don't think this is fixed yet? At least,  I still can't find `burst_inserts_default` in this file.

Reply:
Nevermind, I typo'ed. Thanks for fixing it!